tests/PythonQtTests.cpp

```diff
@@ -193,7 +193,11 @@ void PythonQtTestSlotCalling::testPODSlotCalls()
   QVERIFY(_helper->runScript("if obj.getUInt(42)==42: obj.setPassed();\n"));
   QVERIFY(_helper->runScript("if obj.getShort(-43)==-43: obj.setPassed();\n"));
   QVERIFY(_helper->runScript("if obj.getUShort(43)==43: obj.setPassed();\n"));
+#if (CHAR_MIN + 0)
```

Comment:
If the "signedness" of char isn't fixed, I'm wondering if we should have a separate test for signed char then.
Does PythonQt even support this?

Reply:
I suggest (and this is implemented in the proposed patch) to test for `signed char` iff the implementation is signed, otherwise the test is performed for unsigned value grater than 128 (without loss of generality, I've used 250 in the patch).
If you suggest adding a check for signed char parameter overflow and unsigned char negative/overflow? Please, explain your idea to me, I'll add another check if it can be useful too.